Makefile

```diff
@@ -124,21 +124,21 @@ GOCOVERPKG_ARG := -coverpkg="$(MODULE_ROOT)/common/...,$(MODULE_ROOT)/service/..
 ##### Tools #####
 update-checkers:
 	@printf $(COLOR) "Install/update check tools..."
-	go install golang.org/x/tools/cmd/goimports
-	go install golang.org/x/lint/golint
-	go install honnef.co/go/tools/cmd/staticcheck
-	go install github.com/kisielk/errcheck
-	go install github.com/googleapis/api-linter/cmd/api-linter
-	go install github.com/bufbuild/buf/cmd/buf
+	cd && GO111MODULE=on go get golang.org/x/tools/cmd/goimports
```

Comment:
why there is a `cd && `

Reply:
I need to go to some dir which is outside of `$GOPATH` and doesn't have `go.mod`. `cd $(mktemp -d)` is another option but I don't want to pollute `/tmp` w/o good reason.